DeepNeuralNetworks/DNN.ipynb

```diff
@@ -1,1399 +1,2548 @@
 {
```

Comment:
<p>All this stuff is really useful (and should be) but I think it front loads a lot of math that is gonna put the kids to sleep. Add it to a supplementary materials appendix maybe? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermilab-RENEW/tutorials_2024/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>